tests/test_completion_core.py

```diff
@@ -32,141 +32,3 @@ def test_dict_response(self):
     def test_incomplete_dict_response(self):
         self.assertEqual(parse_completion_response({'items': [], 'isIncomplete': True}), ([], True))
 
-
-class CompletionFormattingTests(unittest.TestCase):
-
-    def test_only_label_item(self):
-        result = format_completion({"label": "asdf"})
-        self.assertEqual(len(result), 2)
-        self.assertEqual("asdf", result[0])
-        self.assertEqual("asdf", result[1])
-
-    def test_prefers_insert_text(self):
```

Comment:
There is a test in test_completions for this:
- test_prefer_insert_text_over_label